python3: can't open file 'massListSorter.py': [Errno 2] No such file or directory
28 0.13679807573413577
Steps: 1000
Beginning distance 0.13679807573413577.
Total steps: 1000
distance: 0.13679807573413577 | 0 time = 0.0 years | 2003 particles | 0 step number |
| 0.0 second | 0.0 minutes.

simAU distance: 0.13679807573413577 | 0.0 time = 0.0 years | 2003 particles | 0 step number |
| 0.0 second | 0.0 minutes | 0.0 hours.

TIME:0.33402864760418643
########################################


The planets collided!Time: 0.33402864760418643


########################################
simAU distance: 0.13679807573413577 | 1.9889065597189395 time = 0.3165443103271652 years | 1902 particles | 1 step number |
| 283.6 second | 4.7 minutes | 0.1 hours.

Done at 283 seconds!
Distance (done): 0.13679807573413577 | 1.9889065597189395 time = 0.3165443103271652 years | 1902 particles | 1000 step number |
| 283.9 second | 4.7 minutes | 0.1 hours.

Masses 0.0030000000000000027 and 0.0.
Ending distance 0.13679807573413577.

Distance 0.13679807573413577 in total took 284 seconds (4.74 minutes, 0.08 hours).
Traceback (most recent call last):
  File "simAUBatch.py", line 616, in <module>
    masslist_txt_append(simAU_masses,f'Masslists/JupiterSimAU{CLargs.date}Batch.txt','simAU','a')
  File "simAUBatch.py", line 187, in masslist_txt_append
    os.chdir("Downloads/Rebound/ReboundResonances/simAU")
FileNotFoundError: [Errno 2] No such file or directory: 'Downloads/Rebound/ReboundResonances/simAU'
slurmstepd: error: *** JOB 602685 ON c52 CANCELLED AT 2021-08-08T19:37:24 ***
